.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,23 @@
+name: Suggest a Feature
+description: Have an idea for something completely new or for improving something that UndertaleModTool has or could benefit from doing or having? Let us know!
```

This sentence reads a little weird due to abundance of `or`s
Maybe 
```suggestion
description: Have an idea for something completely new that UndertaleModTool could benefit from having? Or for improving something that already exists? Let us know!
```